models/user.go

```diff
@@ -945,13 +945,23 @@ func deleteUser(e *xorm.Session, u *User) error {
 	// ***** END: Star *****
 
 	// ***** START: Follow *****
+	followees := make([]*Follow, 0, 10)
```

Comment:
@ethantkoenig fix seems to be merged in xorm, you can now fetch dependency and it should be ok now